src/scripts/clipperUI/components/modeButton.tsx

```diff
@@ -23,18 +26,18 @@ class ModeButtonClass extends ComponentBase<{}, ModeButtonProps> {
 		if (this.props.selected) {
 			className += " selected";
 		}
-
 		let clipMode: string = ClipMode[this.props.myMode];
 		clipMode = clipMode[0].toLowerCase() + clipMode.slice(1);
 		let idName: string = clipMode + "Button";
 
 		return (
-			<a className={className} aria-selected={this.props.selected} role="button"
+			<a className={className} role="option" data-setnameforarrowkeynav={Constants.AriaSet.modeButtonSet}
```

Comment:
See my previous comment on adding this attribute using enableInvoke. This way you don't have to set it explicitly again (which also is resilient against attribute renaming, which this is susceptible to)